code/game/area/Space_Station_13_areas.dm

```diff
@@ -95,6 +95,16 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	. = ..()
 	set_dynamic_lighting()
 
+/area/planet/clown

```

Comment:
These are unused

Reply:
These are rebase memes, I'm honestly not sure if I should even try undoing this stuff or just try to revert everything.